src/Client.php

```diff
@@ -17,23 +20,21 @@
 use Symfony\Component\Validator\Constraints\Url;
 use Symfony\Component\Validator\ConstraintViolationInterface;
 use Symfony\Component\Validator\Validation;
+use Throwable;
 
 class Client
 {
-    const DEFAULT_USER_AGENT = 'Billing PHP Client';
-    const DEFAULT_BACKOFF_RETRIES = 10;
+    private const DEFAULT_USER_AGENT = 'Billing PHP Client';
+    private const DEFAULT_BACKOFF_RETRIES = 10;
+    private const CONNECT_TIMEOUT = 10;
+    private const CONNECT_RETRIES = 0;
+    private const TRANSFER_TIMEOUT = 120;
```

Comment:
Jestli spravne chapu, tak `CONNECT_RETRIES = 0` se chova vlastne jako do ted, takze se to jen udelalo explicitne a ty timeouty jsou nejaky standardni, co pouzivame jinde.

Reply:
ano